### ISSUE TITLE
if the foreinkey can be NULL, how to use DjangoModelField ?

### ISSUE THREAD

**Issue by zhijiasun** [anthony-tresontani/django-adaptors#16]
i think the docuement should be updated. DjangoModelField not mentidoned at the doc, bu instead of ForeignKey. 
if a foreinkey definition can be NULL, how to deal with this secnario ?
maybe DjanogModelField.to_python() can be changed.

**Comment by digiphd** [comment]
I Agree. The documents are lacking. ForeignKey no longer exists, yet DjangoModelField (I assume its replacement) has no documentation at all. I have to revert back to regular row by row csv importing to support ForeignKey fields, as I can't get it to work, and there are no docs.

**Comment by anthony-tresontani** [comment]
Thanks for your comments. Unfortunately, I do not maintain this lib anymore. Nonetheless, I merge pull request when I get one. So if you feel like fixing it, I would be happy to merge.
